tests/ieee802154_hal/Makefile

```diff
@@ -29,13 +29,8 @@ USEMODULE += event_thread_highest
 USEMODULE += event_callback
 USEMODULE += xtimer
 USEMODULE += ieee802154_radio_hal
+USEMODULE += netdev_default
```

Comment:
`openmote-b` defaults to `at86rf215` so you'll have to drop that board for now.

Reply:
done!